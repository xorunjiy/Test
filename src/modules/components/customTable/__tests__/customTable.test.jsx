import React from 'react';
import CustomTable from '../CustomTable.jsx';

const props = {
    rows: [
        {
            email: "zaz11@gmail.com",
            id: 25354,
            status: 0,
            text: "ZZZZZZZ1",
            username: "Zppppa"
        }
    ],
    columnTitles: [
        {
        id: 2,
        name: "username",
        text: "User name"
        }
    ],
    setTaskStatusFormat: () => void 0
}

describe('CustomTable snapshot', () => {
    it('should correctly render', () => {

        const component = renderer(<CustomTable {...props}/>);

        expect(component).toMatchSnapshot();
    });
});