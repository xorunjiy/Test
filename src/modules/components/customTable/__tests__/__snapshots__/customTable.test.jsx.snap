// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomTable snapshot should correctly render 1`] = `
<div
  className=""
>
  <div
    className=""
  >
    <p
      className=""
      id={2}
      name="username"
    >
      User name
    </p>
  </div>
  <div
    className=""
  >
    <div
      className=""
      id={25354}
    >
      <p
        className=""
      >
        Zppppa
      </p>
      <p
        className=""
      >
        zaz11@gmail.com
      </p>
      <p
        className=""
      >
        ZZZZZZZ1
      </p>
      <p
        className=""
      />
    </div>
  </div>
</div>
`;
